Code/Editor/Core/EditorActionsHandler.h

```diff
@@ -85,6 +85,7 @@ class EditorActionsHandler
     void RefreshToolActions();
 
     // View Bookmarks
+    int m_defaultBookmarkCount = 12;
```

Comment:
Thanks for making this change :+1:

Reply:
Thought could this also use `DefaultViewBookmarkCount`? Not a huge deal... better than what was there before 😅

Reply:
It is, but it sets it on initialization.

Reply:
Ah cool thanks! :+1: